src/Parser/Shaman/Restoration/Modules/Features/AlwaysBeCasting.js

```diff
@@ -43,6 +43,31 @@ class AlwaysBeCasting extends CoreAlwaysBeCastingHealing {
     370, // purge
   ];
 
+  get nonHealingTimeSuggestionThresholds() {
```

Comment:
I recommend also using these threshold values in the `suggestions()` method so that you have 1 place you need to update should you ever want to change these. Also try to use the `actual` property there if possible.